internal/pkg/agent/application/upgrade/rollback.go

```diff
@@ -199,3 +219,7 @@ func restartAgent(ctx context.Context, log *logger.Logger) error {
 	close(signal)
 	return nil
 }
+
+type mockAgentServer struct {
+	cproto.UnimplementedElasticAgentControlServer
+}
```

Comment:
Should be deleted?

Reply:
Ah yes, that's a leftover from before I extracted the client and replaced that with a mock. Will delete it